src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -108,18 +108,26 @@ def generate_weighted_strategy_pool():
   # If weighted strategy selection is enabled, there will be a distribution
   # stored in the environment.
   distribution = environment.get_value('STRATEGY_SELECTION_DISTRIBUTION')
+  selection_method = environment.get_value(
+      'STRATEGY_SELECTION_METHOD', default_value='default')
 
   # Otherwise if weighted strategy selection is not enabled (strategy selection
-  # distribution is none) or if we cannot query properly, generate strategy
+  # method is default) or if we cannot query properly, generate strategy
   # pool according to default parameters.
-  if not distribution:
+  if not distribution or selection_method == 'default':
     return generate_default_strategy_pool()
 
+  temperature = 'probability_medium_temperature'
+  if selection_method == 'multi_armed_bandit_high':
+    temperature = 'probability_high_temperature'
+  elif selection_method == 'multi_armed_bandit_low':
+    temperature = 'probability_medium_temperature'
+
   # Change the distribution to a list of named tuples rather than a list of
   # dictionaries so that we can use the randome_weighted_choice function.
   distribution_tuples = [
       StrategyCombination(
-          strategy_name=elem['strategy_name'], probability=elem['probability'])
+          strategy_name=elem['strategy_name'], probability=elem[temperature])
```

Comment:
I thought that this would be the cleanest way to do it, otherwise we have to configure the 'STRATEGY_SELECTION_DISTRIBUTION' variable to be different in each of the 3 experimental cases.